api-server/stack-test-suite/tests/v1/transaction.rs

```diff
@@ -46,6 +46,241 @@ async fn transaction_not_found() {
+                alice_balance = (alice_balance - Amount::from_atoms(random_coin_amount2)).unwrap();
+
+                let alice_tx_out =
+                    TxOutput::Transfer(OutputValue::Coin(alice_balance), alice_destination.clone());
+
+                let bob_tx_out1 = TxOutput::Transfer(
+                    OutputValue::Coin(Amount::from_atoms(random_coin_amount1)),
+                    bob_destination.clone(),
+                );
+
+                let bob_tx_out2 = TxOutput::Transfer(
+                    OutputValue::Coin(Amount::from_atoms(random_coin_amount2)),
+                    bob_destination.clone(),
+                );
+
+                let transaction2 = TransactionBuilder::new()
+                    .add_input(
+                        TxInput::from_utxo(
+                            OutPointSourceId::Transaction(previous_transaction_id),
+                            0,
+                        ),
+                        previous_witness.clone(),
+                    )
+                    .add_output(alice_tx_out.clone())
+                    .add_output(bob_tx_out1.clone())
+                    .add_output(bob_tx_out2.clone())
+                    .build();
+
+                previous_witness = InputWitness::Standard(
+                    StandardInputSignature::produce_uniparty_signature_for_input(
+                        &alice_sk,
+                        SigHashType::try_from(SigHashType::ALL).unwrap(),
+                        alice_destination.clone(),
+                        &transaction2,
+                        &[Some(&previous_tx_out)],
+                        0,
+                    )
+                    .unwrap(),
+                );
+
+                let signed_tx2 = SignedTransaction::new(
+                    transaction2.transaction().clone(),
+                    vec![previous_witness.clone()],
+                )
+                .unwrap();
+
+                let block_id = *tf
+                    .make_block_builder()
+                    .add_transaction(signed_tx1.clone())
+                    .add_transaction(signed_tx2.clone())
+                    .build_and_process()
+                    .unwrap()
+                    .unwrap()
+                    .block_id();
+                let block = tf.block(block_id);
+
+                let transaction_id = signed_tx2.transaction().get_id();
+
+                let utxos = signed_tx2.inputs().iter().map(|inp| match inp {
+                    TxInput::Utxo(outpoint) => {
+                        Some(signed_tx1.outputs()[outpoint.output_index() as usize].clone())
+                    }
+                    TxInput::Account(_) | TxInput::AccountCommand(_, _) => None,
+                });
+
+                let transaction = signed_tx2.transaction();
+
+                let expected_transaction = json!({
+                "block_id": block_id.to_hash().encode_hex::<String>(),
+                "timestamp": block.timestamp().to_string(),
```

Comment:
The formatting here is a bit weird. Is this what rustfmt produces? I would expect the keys to be indented.

Reply:
it is the json!() macro that is killing the formatting